extract_bioheart_genos_demux.py

```diff
@@ -45,17 +44,18 @@
 
 # filter out loci & variant QC
 mt = mt.filter_rows(
-    mt.was_split  # biallelic (exclude multiallelic)
+    ~(mt.was_split)  # biallelic (exclude multiallelic)
```

Comment:
that's cool! I was thinking `mt.filter_rows(mt.was_split == False)`, but that looks cleaner

Reply:
could you maybe just add in a comment about not wanting the previous multiallelics due to sequencing artefacts?